build/ci/win_test.ps1

```diff
@@ -25,6 +51,32 @@ git checkout $REVISION
 Write-Output "Vendor dependencies"
 $env:GOPROXY=$GOPROXY
 tar -xzf ../vendor.tar.gz
+
+Write-Output "Waiting for Docker daemon to start..."
+$maxAttempts = 60  # Wait up to 60 seconds
+$attempt = 0
+
+while ($attempt -lt $maxAttempts) {
+    try {
+        docker version | Out-Null
+        if ($LASTEXITCODE -eq 0) {
+            Write-Output "Docker daemon is ready!"
+            break
+        }
+    }
+    catch {
+        # Ignore errors while waiting
+    }
+    
+    Start-Sleep -Seconds 1
+    $attempt++
+    Write-Output "." -NoNewline
```

Comment:
I'm not sure `-NoNewLine` is supported by Write-Output command, maybe https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-host?view=powershell-7.5 ? 

(didn't check the version we use here) ignore me if it's wrong